### PR TITLE
Created a basic front-end for the website...

### PR DESCRIPTION
**Pull request by AnandKumar22**
It's just the starting one ........ still under constructions.....a lot more toimplement

### CODE REVIEW

**Comment by madan96** [comment]
@AnandKumar22 I'm unable to find the html files. Have you added them to the tar file?

File: README.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Install all the dependencies using `pip install -r requirements.txt` before usin
    * mechanize
    * tabulate
    * fpdf
+   * BeautifulSoup4
+   * requests
+   (The last 2 are optional as they may be reqired as in few cases or may not be required)
```
Comment (madan96) [inline README.md]: There is no need to mention that they are optional.

File: README.md [hunk 2]
```diff
@@ -68,7 +71,8 @@ Currently supported:
 - [x] Print output in `Prettytable` format
 - [ ] Allow search for Entertainment Channels
 - [x] Provide option to save details as PDF
-- [ ] Send notification to the user
+- [ ] Create an account of the user
```
Comment (madan96) [inline README.md:74]: What will be the purpose of creating a user account?

Comment (AnandKumar22) [reply README.md:74]: If we would have a user account then through email or other platform it would be easy to send the notification to the user.

Comment (AnandKumar22) [reply README.md:74]: Yeah ofcourse......I have attached the html file with the name index.html......

Comment (AnandKumar22) [reply README.md:74]: Also regarding the README file I have added 'BeautifulSoup4' and 'requests'....in dependencies.....these were required when I tried to run main.py ..

**Comment by madan96** [comment]
@AnandKumar22 README.md is alright. But you don't have to add the html files to tar.gz package. Push them as separate files. The tar.gz package should only contain the code for backend so that it can be uploaded to the PyPI repos online.

File: .travis.yml [hunk 3]
```diff
@@ -3,9 +3,15 @@ cache: pip
 python:
   - 2.6
   - 2.7
+  # PyPy versions
+  - "pypy"
+  - "pypy"  # PyPy2 2.5.0
+  - "pypy3" # Pypy3 2.4.0
+  - "pypy-5.3.1"
```
Comment (madan96) [inline .travis.yml:10]: Not needed right now. There are other improvements that need to be made with this file but not this one.

**Comment by madan96** [comment]
Great work on the website! Have you made the front end by yourself or used some template.

Also, update the travis.yml file as suggested. And squash all these commits into one.

**Comment by AnandKumar22** [comment]
I made it completely by myself.......
Please do suggest from your side as well...if any new implementation you think should be there...

**Comment by madan96** [comment]
You should squash the commits into one single commit. Also, resolve the merge conflict.

**Comment by madan96** [comment]
Hey, @AnandKumar22 ! As I said before can you please squash the commits into 1-2 commits so I can merge this PR. Ping me if you need any help.

**Pull request closed without merging**